### PR TITLE
Add deprecation badge and disclaimer

### PR DESCRIPTION
**Pull request by guerreirobeatriz**
#### What problem is this solving?

Added badge and disclaimer regarding the Product Price deprecation.

### CODE REVIEW

File: docs/ProductPrice.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 📢 Use this project, [contribute](https://github.com/vtex-apps/store-components) to it or open issues to help evolve it using [Store Discussion](https://github.com/vtex-apps/store-discussion). 
 
-# Product Price
+# Product Price ![https://img.shields.io/badge/-Deprecated-red](https://img.shields.io/badge/-Deprecated-red)
+
+:warning: **The Product Price block has been deprecated in favor of the [Product Price](https://vtex.io/docs/components/all/vtex.product-price@1.3.0/) app**. Although support for this block is still granted, we strongly recommend you to update your store theme with the Product Price's blocks in order to keep up with the component's evolution.
```
Comment (victorhmp) [inline docs/ProductPrice.md:5]: Better not to fix the app version in the documentation URL :)

```suggestion
:warning: **The Product Price block has been deprecated in favor of the [Product Price](https://vtex.io/docs/components/all/vtex.product-price/) app**. Although support for this block is still granted, we strongly recommend you to update your store theme with the Product Price's blocks in order to keep up with the component's evolution.
```

Comment (Klynger) [inline docs/ProductPrice.md:5]:
```suggestion
:warning: **The Product Price block has been deprecated in favor of the [Product Price](https://vtex.io/docs/components/all/vtex.product-price/) app**. Although support for this block is still granted, we strongly recommend you to update your store theme with the Product Price's blocks in order to keep up with the component's evolution.
```

Comment (Klynger) [reply docs/ProductPrice.md:5]: > Better not to fix the app version in the documentation URL :)

Ué

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Fixed
+
+- Fixed Product Price block documentation by addind a deprecation badge and disclaimer.
```
Comment (victorhmp) [inline CHANGELOG.md:12]:
```suggestion
- Fixed Product Price block documentation by adding a deprecation badge and disclaimer.
```

**Pull request merged**